README.md

```diff
@@ -33,28 +33,42 @@ reStructuredText Docstring Format is the standard. Here is an example:
 
 ### How to update the documentation:
 
-To update the docs branch, checkout the gh-pages branch. Make sure it is up to
-date, then clone the dataprofiler from the feature branch you want in order to
-update the documentation (probably main).
+1. Either with an existing clone of `capitalone/dataprofiler` or clone the `capitalone/dataprofiler` reposotory to your local computer with the following command: 
+```bash
+git clone https://github.com/capitalone/DataProfiler dataprofiler
```

Comment:
I believe it was intentional to use the feature branch. Otherwise, an error occurred when you swapped branches or if you want to use a brand separate from main.